src/diffusers/models/modeling_utils.py

```diff
@@ -183,7 +182,7 @@ def load(module: torch.nn.Module, prefix: str = ""):
     return error_msgs
 
 
-class ModelMixin(torch.nn.Module, PushToHubMixin):
+class ModelMixin(torch.nn.Module, PushToHubMixin, PeftAdapterMixin):
```

Comment:
Will remove from `ModelMixin` and add the separate model classes, instead, as discussed.